src/main/java/com/spotinst/sdkjava/model/converters/oceanCD/OceanCDConverter.java

```diff
@@ -196,6 +196,400 @@ public static ApiStepsSetCanaryScale toDal(StepsSetCanaryScale src) {
         return apiStepsSetCanaryScale;
     }
 
+    public static ApiRolloutSpec toDal(RolloutSpec src) {
```

Comment:
Can we have separate convertor class for each type - strategy, RolloutSpec etc.?

Reply:
separated the converter class and tested